mock-echo-app/src/cmr/mock_echo/api/urs.clj

```diff
@@ -46,6 +46,16 @@
        :headers {"content-type" mt/xml}})
     {:status 404 :body "Not found.\n"}))
 
+(defn get-user-info
+  "Returns mock URS info for a user"
+  [user-id]
+  (if-not (= "null" user-id)
+    {:status 200 :body {:first_name "mock"
+                        :last_name "name"
+                        :uid user-id
+                        :email_address (str user-id"@example.com")}}
+    {:status 500 :body "<html></Stub-HTML-Page></html>"}))
```

Comment:
what is that syntax, xhtml allows for <br /> is that something your after?

Reply:
The reason I give a bogus html response like this is that, if you hit this URS endpoint with a non-existent username (`/api/users/foobar_user`) it throws a 500, and the body is just a huge chunk of HTML. So, this was just meant to mock the actual response a bit more accurately - if the status _is_ 500, we do no processing with the body regardless.

Reply:
Your point stands, it's a bad HTML response. However, the `body` in this response could be removed entirely, there would be no change in functionality. Purely a formality.

Reply:
yea, but that syntax is not valid html or xhtml, xml, or anything, <Stub-HTML-PAGE /> at least looks like it is possible.

Reply:
Fair point, updated